### DISCUSSION TITLE
Odd wget error when attempting to convert a .ckpt file

### DISCUSSION THREAD

**Discussion by Eorlanas** [azuritecoin/OnnxDiffusersUI#17] [Q&A] [answered]
I'm having some issues using the ckpt to diffusers script explained in the main page - using it results in a "Unable to establish SSL connection" error, followed by a "some weights of the model checkpoint at openai/clip-vit-large-patch14 were not used" error that then lists what I suspect is every single weight. The script then ends after throwing some more errors about missing keys in state_dict, producing nothing as a result

**Comment by azuritecoin** [comment, marked as answer]
openai/clip-vit-large-patch14 is a CLIP based computer vision classifier. It's not directly compatible with the Stable Diffusion setup that this UI uses.

**Reply by Eorlanas** [reply]
Oof, that's a wrench in the plans
Thanks for the info and the tool nonetheless!
